index.qmd

```diff
@@ -1,39 +1,114 @@
 ---
-title: "Seedcase Project"
+title: "Welcome to the Seedcase Project"
 subtitle: A framework for an open and scalable infrastructure for health data
+toc: false
+sidebar: false
+css: styles.css
 about:
   id: hero-heading
-  template: solana
-  image: /images/logo-full-color.png
-  image-width: 20em 
+  template: marquee
   links: 
-    - icon: cone-striped
-      text: "Software (incomplete)"
-      href: "https://github.com/seedcase-project/seedcase-sprout"
-    - icon: info-circle
-      text: "About"
-      href: "https://seedcase-project.org/about/"
-    - icon: people
-      text: "Community"
-      href: "https://community.seedcase-project.org/"
     - icon: diagram-2
-      text: "Design"
+      text: "Design documentation"
       href: "https://design.seedcase-project.org/"
-    - icon: signpost-2
-      text: "Decisions"
-      href: "https://decisions.seedcase-project.org"
 ---
 
 ::: {#hero-heading}
-We believe that by developing high-quality software and providing
-training on activities and tasks related to building, managing, and
-sharing health data, the science and research on health and well-being
-can become better, ultimately helping prevent or manage diseases like
-diabetes.
+The Seedcase Project is a value-driven initiative focused on open
+science and open-source software. We believe that developing open-source
+software and offering researchers training in technical skills and
+knowledge can improve health and well-being research. This includes
+tasks related to building, managing, and sharing health data.
 :::
 
-### Specific goals
+## Our goals
 
-{{< include includes/_goals.qmd >}}
+The Seedcase Project is more than software, it is also about education,
+training, and the code itself.
+
+::::::: {.grid .wider-grid}
+::: {.g-col-6 .g-col-md-3 .landing-page-card}
+### Develop open-source software for health data
+
+Software that is open-source and targeted to diverse users, like health
+research organizations and small-to-medium sized companies.
+:::
+
+::: {.g-col-6 .g-col-md-3 .landing-page-card}
+### Create training material and run workshops
+
+Specifically training on technical skills such as software and data
+engineering, while following open scientific practices.
+:::
+
+::: {.g-col-6 .g-col-md-3 .landing-page-card}
+### Build and nurture an inclusive community
+
+A community that is open and inclusive that shares knowledge, learnings,
+and better practices in research data engineering and software
+engineering.
+:::
+
+::: {.g-col-6 .g-col-md-3 .landing-page-card}
+### Lead by example
+
+By building this project as fully open as possible and applying the same
+principles we teach and advocate for.
+:::
+:::::::
 
 <!-- TODO: List overview of purposes/principles here? -->
+
+::::::: landing-page-banner
+:::::: landing-page-banner-content
+## Software products
+
+::::: grid
+::: g-col-8
```

Comment:
@lwjohnst86 Why is this without `{}`  in contrast to e.g., L29-30?

Reply:
@signekb when only one class is provided, the formatter removes the `{}`. It's not me!! 😛

Reply:
Very weird!